packages/sdks/core-js-sdk/src/sdk/flow/types.ts

```diff
@@ -6,3 +6,9 @@ type JSONSerializable =
   | Array<JSONSerializable>;
 
 export type FlowInput = Record<string, JSONSerializable>;
+
+export type FlowVersions = {
+  version: number;
+  componentsVersion: string;
+  versions: Record<string, number>;
```

Comment:
maybe worth to name the attributes here `currentVersion` and `otherVersions`? your choice